src/MongoDB.Driver.Core/Core/Servers/ServerMonitor.cs

```diff
@@ -93,7 +93,7 @@ public void Initialize()
 
         public void Invalidate()
         {
-            SetDescriptionIfChanged(_baseDescription);
+            SetDescription(_baseDescription.With(lastUpdateTimestamp: DateTime.UtcNow));
```

Comment:
Are these changes to the lastUpdateTimestamp strictly necessary to actually fix the issue at hand?  I don't seen any new places where this field is used, and AFAIK the only current usage only applies to connected servers.

Reply:
You are correct, the changes to the lastUpdateTimestamp aren't strictly needed now that we're no longer checking the timeStamps when applying ServerDescription updates. The changes are minor correctness changes that simply make debugging easier, and I thought it seemed more efficient to simply include the fixes instead of creating a new ticket to address it.  If you'd prefer though, I can revert these changes and file a low priority ticket :)

Reply:
I'll leave it up to you and Robert to determine that.  I just want to make sure it's clear what's strictly necessary.

Reply:
I'm fine with the minor changes to keep the timestamp updated properly.

Reply:
Okay, Robert and I discussed it, and we think it's more efficient to leave the changes in then to create a ticket, so we're moving forward!